client/src/inside/components/mar/__mocks__/mar-entity.js

```diff
@@ -0,0 +1,95 @@
+import { validTimeStr, validDayStr } from '../../../../helpers/ehr-utils'
+import MedOrder from '../med-order'
+
+
+const ERR_WHO = 'Must provide who administered'
+const ERR_WHEN = 'Time must be 24:00 hour format'
+const ERR_DAY = 'Day must be one of 0,1,2,3,...'
+const ERR_EMPTY_MEDS = 'Medication list can not be empty'
+const DEFAULT_TIME = '0:00'
+const DEFAULT_DAY = 0
+
```

Comment:
Why do we need to replicate so much code from the original?  This will create maintenance challenges.  Can we reuse the original more?

Reply:
Well, this is a bit odd. I will explain below what led me into implementing it this way and then follow up on how the implementation changed (I'm not sure why). 

I thought about implementing this mock class because of an import error regarding the error, which was apparently the same one we ran into while writing the other tests: when trying to import and instantiate the class, it is of type function instead of a class... therefore instantiating it threw an error  (something like `cannot instantiate @@iterator`). This was what led me to implement the mock class and this __mock__ replication of `mar-entity`. However, as I was writing this comment, I tried to undo this and try a direct import of `mar-entity` and, as it turned out, that error was no longer thrown. I am not sure as to why... but this is how it happened. Therefore, I will commit these changes and remove the mock altogether... if you have any ideas of why this happened, I would love to hear more about them. 

As always, thank you very much for your feedback!

Reply:
That's good.  Sorry have no idea but strange things like this happen to me too